### ISSUE TITLE
Forge search requests fail when executing behind a proxy

### ISSUE THREAD

**Issue by azbarcea** [hsnodgrass/pufctl#2]
The following ENVs were exported:

```
https_proxy
http_proxy
use_proxies=yes
```

The following error is displayed:
``` 
sudo-sh[root@dlv-eu4-a046]# pufctl search forge puppet_agent
PUFCTL - ERROR: 2020/10/12 10:46:22 Search failed with error:  Search failed: forgeapi: List failed: forgeapi: 2020-10-12 10:46:22.774120684 -0400 EDT m=+5.006676982: HTTP GET request failed. URL: https://forgeapi-cdn.puppet.com/v3/modules?limit=5&query=puppet_agent&sort_by=rank, Headers: map[string]string{​​​​​​​"User-Agent":"pufctl/v0.1.0"}​​​​​​​, Error: &url.Error{​​​​​​​Op:"Get", URL:"https://forgeapi-cdn.puppet.com/v3/modules?limit=5&query=puppet_agent&sort_by=rank", Err:(*http.httpError)(0xc000229660)}​​​​​​​
```

Normally it should use the ENV proxy exported above.

kernel: 4.12.14-122.20-default x86_64 GNU/Linux

**Comment by hsnodgrass** [comment]
@azbarcea This should be fixed with `v0.1.1` which is currently building. Please download the binary for `v0.1.1` and let me know if that works for you.

**Comment by hsnodgrass** [comment]
Verified working correctly with `v0.1.1`
